iOS/testbed/Python.xcframework/Info.plist

```diff
@@ -4,6 +4,23 @@
 <dict>
 	<key>AvailableLibraries</key>
 	<array>
+		<dict>
+			<key>BinaryPath</key>
+			<string>Python.framework/Versions/3.14/Python</string>
```

Comment:
And *this* is why a versioned framework isn't a good idea.

At the very least, this should be pointing at `Latest`.